.github/ISSUE_TEMPLATE/custom.md

```diff
@@ -0,0 +1,52 @@
+---
+name: Custom issue template
+about: Describe this issue template's purpose here.
+title: ''
+labels: ''
+assignees: ''
```

Comment:
If we're going to add metadata, it should have a useful name and description at least. Since this template is for bug reports, I suggest the name and description should reflect that.

Perhaps we could have two separate templates, one for bug reports and one for feature requests or other. If we would prefer that users use other channels for feature requests we can include that information in the feature request template. There are metadata fields for contact links for example. See https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates.